.github/workflows/eamxx_default_files.yml

```
@@ -0,0 +1,55 @@
+name: inputdata
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: '00 00 * * *'
+  workflow_dispatch:
+
+jobs:
+  scream-defaults:
+    runs-on: ubuntu-latest
+    outputs:
+      event_name: ${{ github.event_name }}
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@v4
+        with:
+          show-progress: false
+          submodules: false
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v4.7.0
+        with:
+          python-version: "3.11"
+      - name: Run unit tests
+        working-directory: components/eamxx/cime_config/
+        run: |
+          python -m unittest tests/eamxx_default_files.py -v
+
+  notify-scream-defaults:
+    needs: scream-defaults
+    if: ${{ failure() && needs.scream-defaults.outputs.event_name != 'pull_request' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create issue
+        run: |
+          previous_issue_number=$(gh issue list \
+          --label "$LABELS" \
+          --json number \
+          --jq '.[0].number')
+          gh issue edit "$previous_issue_number" \
```

Two things:
- we may also do `gh issue comment`, so we keep track of how the missing list evolved (assuming nobody fixed it in the meantime) rather than overwriting. Not sure if it'd help, but can't hurt...
- we still need ot check if the issue exists, otherwise I guess this may fail? something like
```suggestion
if [ -z "$previous_issue_number" ]; then
  gh issue open <labels, assignees, body...>
else
  gh issue comment <new comment>
fi
```